<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Unos</class>
 <widget class="QDialog" name="Unos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unos</string>
  </property>
  <widget class="QLabel" name="sifraLb">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Šifra</string>
   </property>
  </widget>
  <widget class="QLabel" name="mlb">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mesto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="izlaz">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IZAĐI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="obrisi">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OBRIŠI</string>
   </property>
  </widget>
  <widget class="QLabel" name="katLbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Datum</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Izbor akcije</string>
   </property>
   <widget class="QRadioButton" name="upis">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Upis</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="brisanje">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Brisanje</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="upisi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UPIŠI</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSifra">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMesto">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>6</month>
     <day>5</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2030</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
